FCCee/CLD/compact/CLD_o3_v01/InnerTracker_o2_v07.xml

```diff
@@ -17,13 +17,15 @@
         <constant name="InnerTracker_Endcap_z_5" value="1775*mm" />
         <constant name="InnerTracker_Endcap_z_6" value="1990*mm" />
 
+        <!-- Along with the z position change we would need to adapt the inner radius accordingly
+             to stay close to the 150 mrad coverage -->
         <constant name="InnerTracker_Endcap_radius_0" value="79.5*mm" />
-        <constant name="InnerTracker_Endcap_radius_1" value="123.5*mm" />
-        <constant name="InnerTracker_Endcap_radius_2" value="165*mm" />
-        <constant name="InnerTracker_Endcap_radius_3" value="207.5*mm" />
-        <constant name="InnerTracker_Endcap_radius_4" value="249.5*mm" />
-        <constant name="InnerTracker_Endcap_radius_5" value="293*mm" />
-        <constant name="InnerTracker_Endcap_radius_6" value="330*mm" />
+        <constant name="InnerTracker_Endcap_radius_1" value="117*mm" />
+        <constant name="InnerTracker_Endcap_radius_2" value="154.6*mm" />
+        <constant name="InnerTracker_Endcap_radius_3" value="192.4*mm" />
+        <constant name="InnerTracker_Endcap_radius_4" value="230.1*mm" />
+        <constant name="InnerTracker_Endcap_radius_5" value="268*mm" />
+        <constant name="InnerTracker_Endcap_radius_6" value="300.5*mm" />
```

Comment:
Should'nt this change also be propagated to the sensor size?

Reply:
Hi, I have checked the endcaps (or disks) and it seems the change in radius makes the sensors to be placed properly, without leaving holes

Reply:
One "sub-module" of a InnerTracker Module is 15.1 mm, modules are always multiples of these 15.1 mm. So reducing the radius by 30 mm leaves quite a larger gap with respect to before. Which kind of makes me afraid one would have to change the modules used. So for example instead of `InnerTrackerEndcapModule_12x8_Out` one would use a `InnerTrackerEndcapModule_12x10_Out`, which would be 30.2 mm longer. Except one would have to check that this really fits.

I am not sure if the positions before where somehow thought out that these 15.1 mm dimensions always kind of end up at the same radius, or this was just random'ish.

Reply:
Shall I try to replace the module size and run overlap check?

Reply:
I think this needs more than just passing the overlap check, but a bit of designing.

Reply:
Hi @atolosadelgado 
How is the progress with checking the numbers and gaps there are now compared to before?

Reply:
I just had time to run with Geantinos to assess your claim of the lack of angular coverage, but not time after Paestum to play with it. I will spend time on this as soon as I implement some new features to ARC.